libraries/splunk_password.rb

```diff
@@ -8,7 +8,7 @@
 # Module contains different functions to encrypt and decrypt splunk passwords
 module CernerSplunk
   # Encrypts the password before writing into config files. As of now all the passwords
-  # needs to be XORed except for the sslKeyFilePassword. The boolean
+  # needs to be XORed except for the sslKeyFilePassword / sslPassword. The boolean
```

Comment:
Huh... how about that, I missed that too. You're right though we should kill references to deprecated settings... We'll throw it in with the next cleanup. I also want to get rid of all the constants that yell when rspec runs: https://github.com/acharlieh/cerner_splunk/tree/sweeping